Documentation/trying-out-rkt.md

```diff
@@ -1,13 +1,23 @@
-# Trying out rkt 
+# Trying out rkt
 
-This guide provides a short introduction to trying out rkt.
-For a more in-depth guide of a full end-to-end workflow of building an application and running it using rkt, check out the [getting-started-guide](getting-started-guide.md)
+This document introduces the basics of getting rkt and running a container with it. For a more in-depth guide to building application containers and running them with rkt, check out the [getting started guide][getstart].
+
+Giving rkt a spin takes just a few basic steps, detailed below.
+
+1. Install rkt:
+  * [CoreOS Linux][run-coreos] comes with rkt installed and configured. You can skip straight to [running a container with rkt][id-rkt-run-by-name].
+  * On other Linux distributions, [grab the latest rkt binary][id-lin-rkt-bin], or [the distribution's rkt package][id-lin-rkt-pkg].
+  * On Mac or Windows, you can [use a Vagrant virtual machine to run rkt][id-vagrant-rkt]
+2. [Configure rkt][id-config-rkt]: Optional steps that make it simpler to experiment with rkt.
+3. [Download][id-rkt-fetch] and [run a container with rkt][id-rkt-run].
 
 ## Using rkt on Linux
 
-rkt consists of a single CLI tool and can run on different platforms. The primary target platform currently is amd64 Linux. A modern kernel is required but there should be no other system dependencies.
+rkt is written in Go and can be compiled for several CPU architectures. The rkt project distributes binaries for amd64. These rkt binaries will run on any modern Linux amd64 kernel.
```

Comment:
https://github.com/coreos/rkt/pull/2758 (arm64 support) is merged, so it needs an update.